### ISSUE TITLE
Z+- arrows wrong?

### ISSUE THREAD

**Issue by neturmel** [ballaswag/guppyscreen#28]
In my opinion, the arrow icons for z+ and z- movement are reversed, z+ moves the nozzle up and z- down.  The arrows are only correct if you see it as z offset.  But maybe that is only me.

**Comment by evgarthub** [comment]
I noticed that too. If you open Fluidd/Mainsail next to gappyscreen with z-offset controls. Plus and minus signs are opposite relative to the arrow direction on it.
I wanted to double check what they actually do, but didn't get chance to do proper test.

**Comment by ballaswag** [comment]
This is on the homing panel or the fine tune panel?

**Comment by evgarthub** [comment]
Fine tune while active print is running

**Comment by ballaswag** [comment]
Will flip it on next release

**Comment by evgarthub** [comment]
I'm in guppyscreen v0.0.21-beta and it's still opposite of what I see in mainsail.
![Screenshot_20240217-193405](https://github.com/ballaswag/guppyscreen/assets/23404448/7a7c4904-981a-4cf1-b9f6-8fa846b15a6c)
![PXL_20240217_173426990 MP](https://github.com/ballaswag/guppyscreen/assets/23404448/bae26a8c-8636-4c49-977d-7f4c21f7d184)

**Comment by ballaswag** [comment]
Thanks for the pictures. Will get to it in next release, just had to recreate the c array assets after renaming these 2 icons.

**Comment by spitfire** [comment]
@ballaswag also some movement icons on the movement screen are reversed (as compared to what printer does. I think it's both Z an Y axes

**Comment by henrivdr** [comment]
> @ballaswag also some movement icons on the movement screen are reversed (as compared to what printer does. I think it's both Z an Y axes

I agree.  @ballaswag Please add an option in the Config screen to invert the direction(s), like Fluid does with "Invert X/Y/Z control"

**Comment by Orel-A** [comment]
A user of the V3 KE here.
Why there is an option in the settings to invert the Z icons instead of it being fixed by default?
Z- is down and Z+ is up, as it's in Klipper. Is it not?

**Comment by spitfire** [comment]
> A user of a V3 KE here. Why there is an option in the settings to invert the Z icons instead of it being fixed by default? Z- is down and Z+ is up, as it's in Klipper. Is it not?

it's not if you have a K1 where the hotbed moves, and not your hotend, so I could say the same about your statement ;)

**Comment by Orel-A** [comment]
Z+ is distancing the nozzle from the bed and Z- is making it closer, is it not?
Then the icon is wrong, the icon shows a dot and up, meaning distancing the hotend from the bed.
Aren't there better icons for this?

**Comment by spitfire** [comment]
> Z+ is distancing the nozzle from the bed and Z- is making it closer, is it not? Then the icon is wrong, the icon shows a dot and up, meaning distancing the hotend from the bed. Aren't there better icons for this?

have you read my previous comment? On KE (which I had) the hotted moves down on Z-, on K1 bed moves up.

@ballaswag is is possible to display it the visually correct way depending on the machine it's running on?
